src/Tools/Source/FakeSign/FakeSign.csproj

```diff
@@ -1,38 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Won't this cause an internal build break?  I thought that it referenced this too.